src/config.ts

```diff
@@ -17,17 +24,43 @@ interface FileChangesConfig {
 
 const versionBumpChange = ['-  "version": "', '+  "version": "'];
 
-const packageManagerConfig: Record<string, FileChanges> = {
-	npm: {
-		'package.json': versionBumpChange,
-		'package-lock.json': versionBumpChange,
-	},
-	yarn: {
-		'package.json': versionBumpChange,
-	},
+const npmLockfileChanges = (npmLockfileVersion: number): string[] => {
+	switch (npmLockfileVersion) {
+		case 1: {
+			return versionBumpChange;
+		}
+		case 2: {
+			/*
+			The v2 npm lockfile contains the version of a package twice:
+				- `.version`
+			  	- `.packages."".version`
+
+			As a result, we expect more changes.
+			 */
+			return [...versionBumpChange, ...versionBumpChange];
+		}
+		default:
```

Comment:
There is a [v3 lockfile](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion), however I don't expect we'll be seeing this for a while as it requires npm v6 to be fully deprecated.